src/workerd/api/wpt/urlpattern-test.ts

```diff
@@ -17,151 +17,21 @@ export default {
     comment: 'URLPattern.compareComponent is not part of the URLPattern spec',
     skipAllTests: true,
   },
-  'urlpattern-hasregexpgroups-tests.js': {
-    comment: 'urlpattern implementation will soon be replaced with ada-url',
-    expectedFailures: [
-      // Each of these *ought* to pass. They are included here because we
-      // know they currently do not. Each needs to be investigated.
-      '', // This file consists of one unnamed subtest
-    ],
-  },
+  'urlpattern-hasregexpgroups-tests.js': {},
   'urlpattern-hasregexpgroups.any.js': {},
   'urlpattern.any.js': {},
   'urlpattern.https.any.js': {},
   'urlpatterntests.js': {
-    comment: 'urlpattern implementation will soon be replaced with ada-url',
+    comment: 'skipping invalid tests',
     expectedFailures: [
-      // Each of these *ought* to pass. They are included here because we
```

Comment:
I assume that this comment is no longer needed since the tests themselves are invalid here.